.eslintrc.json

```diff
@@ -11,16 +11,13 @@
     "@tinymce/no-enums-in-export-specifier": "off",
     "@tinymce/no-unimported-promise": "off",
     "@typescript-eslint/ban-ts-ignore": "off", // this will be added to the plugin
-    "@typescript-eslint/camelcase": "off",
-    "@typescript-eslint/consistent-type-definitions": "off",
-    "@typescript-eslint/explicit-member-accessibility": "off",
+    "@typescript-eslint/camelcase": "off", // leave off
```

Comment:
Comments aren't valid in JSON, so we shouldn't be using them. If we want comments, we should probably convert this to the JS configuration variant.

Reply:
Pretty sure it's JSONC in this case (as I know vscode complains when you add comments to the package.json).

Reply:
Looks like it, it does mention in the docs it allows comments [1]. Seems weird that it's not using the `jsonc` extension then instead.

[1] https://eslint.org/docs/user-guide/configuring#configuration-file-formats

Reply:
Yeah, vs code identifies any file you name .eslintrc.json as "JSON with Comments"

Reply:
I see this in other situations too, supporting JSONC but sticking to the `.json` file format, I guess because people won't know what a .jsonc file is? 🤷‍♂ 

If vscode didn't have an automatic override it's very easy to create one